### DISCUSSION TITLE
Tuya trial account and Tuya-Local

### DISCUSSION THREAD

**Discussion by pdbernardoii** [make-all/tuya-local#2271] [General]
Hello and thank you very much for any help you can give me.

If my Tuya IoT trial account expires, will devices continue to work on Tuya-Local?

I find the Tuya-Local much more practical than the local Tuya, and I have several devices working perfectly, an energy meter/protector, a Nedis 20L dehumidifier, and a star projector.

I'm afraid that when the Tuya account expires I will have no local access to the devices and I will have to create another account and do it all over again

**Comment by make-all** [comment]
You don't need a Tuya developer account to use the QR code method of adding new devices. A normal app account is enough.

**Reply by pdbernardoii** [reply]
thank you very much
